extension/lib/emailjs/emailjs-mime-codec.js

```diff
@@ -768,7 +768,7 @@
                     // unencoded lines: {name}*{part}
                     // if any line needs to be encoded then the first line (part==0) is always encoded
                     key: key + '*' + i + (item.encoded ? '*' : ''),
-                    value: /[\s";=]/.test(item.line) ? '"' + item.line + '"' : item.line
+                    value: /[\s";=']/.test(item.line) ? '"' + item.line + '"' : item.line,
```

Comment:
Do you agree with this approach @sosnovsky?
AFAIK as per the RC standard, single quote should be enclosed in double quotes.

With this way, current issue will be resolved but unit tests are complaining about this change and I need to update unit test.

Reply:
Approach is ok for me, issue is fixed with it 👍